### ISSUE TITLE
Error when building for android without specifying Android.Architectures

### ISSUE THREAD

**Issue by AndrewEQ** [fuse-open/uno#214]
I think we need to set reasonable default architecture values if Android.Architectures is not defined:

```
ERROR: Cannot process request because the process (1380) has exited. (pass --trace for stack trace)
adb: failed to install Test.apk: Failure [INSTALL_FAILED_NO_MATCHING_ABIS: Failed to extract native libraries, res=-113]
```

**Comment by mortend** [comment]
I've seen several users having the same issue. It looks like many devices don't support `arm64-v8a` (the new default) and users need to edit unoproj to go back to `armeabi-v7a` (the old default).

I'm considering reverting back to `armeabi-v7a` as the default ABI when building for Debug to solve this problem.

**Comment by mortend** [comment]
Fixed in v1.12.2
